src/java/org/apache/cassandra/schema/MigrationManager.java

```diff
@@ -40,6 +40,7 @@
 import org.apache.cassandra.net.MessagingService;
 import org.apache.cassandra.schema.Keyspaces.KeyspacesDiff;
 import org.apache.cassandra.utils.FBUtilities;
+import org.assertj.core.util.VisibleForTesting;
```

Comment:
don't we normally use the guava one?

Reply:
True; switched to guava one in one of the commits. Thanks for spotting!